src/openloco/windows/music_selection.cpp

```diff
@@ -0,0 +1,236 @@
+#include "../config.h"
+#include "../graphics/colours.h"
+#include "../graphics/image_ids.h"
+#include "../interop/interop.hpp"
+#include "../localisation/string_ids.h"
+#include "../objects/interface_skin_object.h"
+#include "../objects/objectmgr.h"
+#include "../openloco.h"
+#include "../ui/WindowManager.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::windows::music_selection
+{
+    constexpr uint16_t ww = 360;
+    constexpr uint16_t wh = 238;
+
+    static const uint8_t rowHeight = 12;
+    static const uint8_t numTracks = 29;
```

Comment:
Add constant to header (or introduce getter) and use in audio.cpp (already as `num_music_tracks`) and options.cpp